### DISCUSSION TITLE
Bundler Promo ...

### DISCUSSION THREAD

**Discussion by TiddlyTweeter** [wikilabs/plugins#112] [General]
Just an advisory.

I posted a promo about the Bundler for serious use for maintaining wikis here: In [
The problem with “custom solutions”](https://talk.tiddlywiki.org/t/the-problem-with-custom-solutions/1930/22?) I wrote ... 

> In brief, the Bundler …
> 
> Let’s you add comments in the normal Bundles. This is great as the source and the comments go together. It is a minimalist approach. That makes it maintainable.
> - Provides a mechanism via the Info Panel of every bundle to see what is in the package, determine if anything is missing, delete any item.
> - For Filter Bundles, it can also pass the filter to Advanced Search for good integration into normal use.
> - Unlike other import/export options it allows you to transport state tiddlers. That is very helpful when your setup/config uses them.
> - Option to permanently record all imports to an Import Bundle. That allows both tracking of what you did and enables easy archiving of all imports to a JSON output. It is editable.
> - Interestingly, the Bundler and any bespoke settings for it can be bundled into a bundle so you can transport the Bundler along with the other transportees.
> 
> Here is an example of a bundle’s contents I have … 
> ![Screenshot 2022-01-06 105709](https://user-images.githubusercontent.com/28830541/148519549-2a1f176b-e915-4092-8395-82da8ecab8c3.jpg)
>

**Comment by pmario** [comment]
Thx ... I've seen the post and had a similar thought, that I should make it more clear, how the export import workflow can be improved.

**Reply by pmario** [reply]
I did like: "Just answering for myself to myself: Get Your Ass In Gear & Record Changes As You Make 'Em!" from your second post ;) ... I think that's very valid. If it isn't done immediately it won't be done. ..
